SketchUp/Geom/PolygonMesh.rb

```diff
@@ -51,20 +51,70 @@ class Geom::PolygonMesh
   def add_point(point)
   end
 
-  # The {#add_polygon} method is used for adding a polygon to a polygon mesh.
-  #
-  # @example
-  #   mesh = Geom::PolygonMesh.new
-  #   point1 = Geom::Point3d.new(0, 1, 2)
-  #   point2 = Geom::Point3d.new(1, 0, 2)
-  #   point3 = Geom::Point3d.new(2, 0, 1)
-  #   index = mesh.add_polygon(point1, point2, point3)
-  #
-  # @param [Array<Integer>] points
-  #   A list of points that represent the
-  #   polygon's vertices.
-  #
-  # @return [Integer] the index of the polygon in the mesh
+  # The {#add_polygon} method is used for adding a {#polygons polygon} to a
+  # {PolygonMesh}. All variations of this method require at least 3 elements
+  # to define a polygon, although more may be given.
+  #
+  #
+  # @overload add_polygon(index, index, index, ...)
+  #   @example
+  #     mesh = Geom::PolygonMesh.new
+  #     # add points to mesh...
+  #     mesh.add_point(Geom::Point3d.new(0, 0, 0))
+  #     mesh.add_point(Geom::Point3d.new(1, 0, 0))
+  #     mesh.add_point(Geom::Point3d.new(1, 1, 0))
+  #     polygon_index = mesh.add_polygon(1, 2, 3)
+  #   @param index [Integer]
+  #     An index of a vertex in the mesh. Remember, mesh indices start at 1.
+  #   @param ... [Integer]
```

Comment:
It does work.  I don't think yard cares about what the  parameter  name is, yard is probably  using a regular expression to look for  white space delimited  characters  although I did not confirm that. 

 It is somewhat unusual to have a method   Such as an add_polygon that does not have a fixed number of parameters.   I wasn't sure how to document that so the three dots is what I came up with.